nominations/models.py

```diff
@@ -150,7 +150,7 @@ def _campaign_issues(self, *args, **kwargs):
         return position
     campaign_issues = property(_campaign_issues)
 
-    def save(self, *args, **kwargs):
+    def save(self, skip_application_save=False, *args, **kwargs):
```

Comment:
I wonder if it would be better to use the existing `**kwargs` param to pass in `skip_application_save`? This way you don't have to change the method signature at all. Although I am still learning how to use `*args` and `**kwargs` and the concept of undefined and countless potential params seems kind of bizarre. What is the point of having a method signature contract if it's also like "by the way feel free to pass in any number of other params too"? In the way you have it now at least it is obvious at a glance that there is a `skip_application_save` param.

Reply:
Yeah, this is a bit confusing, and I ultimately went this route to illustrate to dev that this was indeed a thing.